code/datums/wounds/burns.dm

```diff
@@ -164,18 +165,31 @@
 	. = ..()
 	. += "<div class='ml-3'>"
 
+	. += "Estimated duration of treatment while the healing rate is active: [span_deadsay("[flesh_damage]s")]\n"
+	if(flesh_healing > 0)
+		. += "Positive signs of healing in the flesh damage, rate: [span_green("[flesh_healing]%")]\n"
+	else
+		. += "Flesh damage is not exhibiting signs of healing, rate: [span_brass("[flesh_healing]%")]\n"
+
 	if(infestation <= sanitization && flesh_damage <= flesh_healing)
-		. += "No further treatment required: Burns will heal shortly."
+		. += "[span_green("No further treatment required: Burns will heal shortly.")]"
 	else
-		switch(infestation)
-			if(WOUND_INFECTION_MODERATE to WOUND_INFECTION_SEVERE)
-				. += "Infection Level: Moderate\n"
-			if(WOUND_INFECTION_SEVERE to WOUND_INFECTION_CRITICAL)
-				. += "Infection Level: Severe\n"
-			if(WOUND_INFECTION_CRITICAL to WOUND_INFECTION_SEPTIC)
-				. += "Infection Level: <span class='deadsay'>CRITICAL</span>\n"
-			if(WOUND_INFECTION_SEPTIC to INFINITY)
-				. += "Infection Level: <span class='deadsay'>LOSS IMMINENT</span>\n"
+		if(infestation > 0)
```

Comment:
A visible "current sanitization effect" percent of sanitization/infestation would be useful to judge how effective treatment actually is, since infection is the main bit for burn wounds